### PR TITLE
Use fs_attachment instead of attachment_s3

### PR DESCRIPTION
**Pull request by adrienpeiffer**

### CODE REVIEW

**Comment by sbidoul** [comment]
Yeah, it's a bit more complicated than that, I'm afraid.

File: src/odoo/addons/{{ project_trigram }}_fs_attachment/data/fs_storage.xml.jinja [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <!-- This storage is used in the production environment as default storage
+      for the attachments. Configuration is provided by server environment files.
+    -->
+    <field name="name">Attachements Production</field>
+    <field name="code">fsprod</field>
+    <field name="protocol">s3</field>
+    <field name="directory_path">odoo-{{ project_name }}-prod</field>
+    <field name="options">
+      {
+        "endpoint_url": "ENV_AWS_HOST",
```
Comment (adrienpeiffer) [inline src/odoo/addons/{{ project_trigram }}_fs_attachment/data/fs_storage.xml.jinja:15]: I assume here that we'll be reading environment variables starting with ENV

Comment (sbidoul) [reply src/odoo/addons/{{ project_trigram }}_fs_attachment/data/fs_storage.xml.jinja:15]: I'd suggest `$AWS_HOST`, but yes that's the idea. Looks good.

File: src/odoo/addons/{{ project_trigram }}_fs_attachment/data/fs_storage.xml.jinja [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    <field name="options">
+      {
+        "endpoint_url": "$AWS_HOST",
+        "key": "$AWS_ACCESS_KEY_ID",
+        "secret": "$AWS_SECRET_ACCESS_KEY",
+        "client_kwargs": {
+            "region_name": "$AWS_REGION"
+          }
+      }
+    </field>
```
Comment (sbidoul) [inline src/odoo/addons/{{ project_trigram }}_fs_attachment/data/fs_storage.xml.jinja:63]: We probably need to set more fields here (obscate names, etc).

We also need to set the default backend used for attachments in server_environment_files.

**Review by sbidoul — APPROVED** [review]

**Comment by sbidoul** [comment]
Thanks!

**Pull request merged**